src/cmd/migrate.ts

```diff
@@ -442,8 +404,60 @@ export const preserveIngressControllerConfig = async (
 export const migrate = async (): Promise<boolean> => {
   const d = terminal(`cmd:${cmdName}:migrate`)
   const argv: Arguments = getParsedArgs()
+  if (await pathExists(`${env.ENV_DIR}/env/settings.yaml`)) {
+    d.log('Detected the old values file structure')
+    // TODO perform migration
+    const output = await hf(
+      { fileOpts: `${rootDir}/helmfile.tpl/helmfile-dump-files-old.yaml`, args: 'build' },
+      undefined,
+      env.ENV_DIR,
+    )
+    const oldValues = parse(output.stdout).renderedvalues
+    // TODO migrate team settings
+    const oldTeams = get(oldValues, 'teamConfig', {})
+    Object.keys(oldTeams).forEach((teamName) => {
+      const settingsKeys = [
+        'alerts',
+        'id',
+        'limitRange',
+        'managedMonitoring',
+        'networkPolicy',
+        'oidc',
+        'resourceQuota',
+        'selfService',
+        'password',
+      ]
+      const teamSettings = pick(oldTeams[teamName], settingsKeys)
+      const teamResources = omit(oldTeams[teamName], settingsKeys)
+
+      const newTeam = { ...teamResources, settings: teamSettings }
+      oldTeams[teamName] = newTeam
+    })
+    const users = get(oldValues, 'users', [])
+    users.forEach((user) => {
+      set(user, 'id', user.id || randomUUID())
+    })
+    oldValues.versions = { specVersion: 1 }
+    const teamNames = await getTeamNames(env.ENV_DIR)
+    const secretPaths = await getSchemaSecretsPaths(teamNames)
+    const valuesPublic = omit(oldValues, secretPaths)
+    const valuesSecrets = pick(oldValues, secretPaths)
+
+    // FIXME migrate workloadValues folder and change ApplicationSet !!
```

Comment:
@CasLubbers could you add migration for workloadValues?
Each team should have its own `workloadValues` directory.
It is good to remember that workload values do not need to be processed by core while deploying.

Reply:
done